apps/aecore/src/aec_tx_pool.erl

```diff
@@ -372,11 +372,6 @@ int_get_candidate_fold(Gas, GasLimit, Db, Dbs = #dbs{}, {Txs, Cont},
                        AccountsTree, Height, Acc) when Gas > GasLimit ->
     {RemGas, NewAcc} = fold_txs(Txs, Gas, GasLimit, Db, Dbs,
                                 AccountsTree, Height, Acc),
-        lists:foldl(
-          fun(T, {Gas1, Acc1}) ->
-                  int_get_candidate_(T, Gas1, Db, Dbs, AccountsTree,
```

Comment:
This has side effect in move_to_visited though seems genuine duplicated code.

Reply:
Yes, that is how I noticed (we were adding it to GC twice!)...